htdocs/adherents/list.php

```diff
@@ -902,15 +914,9 @@
 		print '</td>';
 		if (! $i) $totalarray['nbfield']++;
 	}
-	// Action column
-	print '<td class="center">';
-	if ($massactionbutton || $massaction)   // If we are in select mode (massactionbutton defined) or if we have already selected and sent an action ($massaction) defined
-	{
-		$selected=0;
-		if (in_array($obj->rowid, $arrayofselected)) $selected=1;
-		print '<input id="cb'.$obj->rowid.'" class="flat checkforselect" type="checkbox" name="toselect[]" value="'.$obj->rowid.'"'.($selected?' checked="checked"':'').'>';
-	}
-	print '</td>';
+	// Empty line (former Action column)
+	print '<td>&nbsp;</td>';
```

Comment:
Because on he header, there is still the list for the column filters. This canot work correctly if I move it in the first column with the checkbox "select all". So a there is still the last column in the header, I need empty column on each line in order to have nice table.